spec/controllers/v2/sort/security_guides_controller.yaml

```diff
@@ -0,0 +1,161 @@
+:entities:
+  - :factory: :v2_security_guide
+    :title: 'aaab'
+    :version: '0.1.49'
+    :os_major_version: 8
+  - :factory: :v2_security_guide
+    :title: 'aaaa'
+    :version: '0.1.43'
+    :os_major_version: 7
+  - :factory: :v2_security_guide
+    :title: 'aacb'
+    :version: '0.1.47'
+    :os_major_version: 8
+  - :factory: :v2_security_guide
+    :title: 'aaaa'
+    :version: '0.1.44'
+    :os_major_version: 7
+  - :factory: :v2_security_guide
+    :title: 'btfn'
+    :version: '0.1.45'
+    :os_major_version: 7
+  - :factory: :v2_security_guide
+    :title: 'aaaa'
+    :version: '0.1.44'
+    :os_major_version: 8
+:queries:
+  - :sort_by:
+      - 'title'
+    :result: [[1, 3, 5], 0, 2, 4]
+  - :sort_by:
+      - 'title:asc'
+    :result: [[1, 3, 5], 0, 2, 4]
+  - :sort_by:
+      - 'title:desc'
+    :result: [4, 2, 0, [1, 3, 5]]
+  - :sort_by:
+      - 'version'
+    :result: [1, [5, 3], 4, 2, 0]
```

Comment:
@xmicha82 Why is the `[5, 3]` nested here? Is this case documented somewhere?

Reply:
security guide at indexes 3 and 5 have the same version, so they need to be sorted by ID, which is achieved by nesting them, as explained in `spec/support/shared_exaples/sortable_spec.rb:22`

Reply:
Could you add some demonstration in the comments of how this nesting/sorting works?

After reading `spec/support/shared_exaples/sortable_spec.rb:22`, the process still feels magical.